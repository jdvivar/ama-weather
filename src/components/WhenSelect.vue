<template>
  <div class="line --interactive" @click="$store.commit('toggleWhen')">
    <a>{{ this.$store.state.when }}</a>
  </div>
</template>

<script>
export default {
  name: 'when-select'
}
</script>
