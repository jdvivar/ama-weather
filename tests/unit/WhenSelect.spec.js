import { shallowMount } from '@vue/test-utils'
import WhenSelect from '@/components/WeatherInfo'

fdescribe('WhenSelect', () => {
  it('has correct markup', () => {
    const wrapper = shallowMount(WhenSelect)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
