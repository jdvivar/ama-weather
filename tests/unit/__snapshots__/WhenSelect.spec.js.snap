// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhenSelect has correct markup 1`] = `undefined`;
